plugin/face-mask-filter.h

```diff
@@ -238,6 +238,11 @@ namespace Plugin {
 			smll::DetectionResults		faces;
 			smll::TriangulationResult	triangulation;
 			TimeStamp					timestamp;
+			TimeStamp					lastActualTimetamp;
+			TimeStamp					renderTimetamp;
+			smll::ProcessedResults		processedFrameResults;
```

Comment:
Can you change timetamp to timestamp :)?

Reply:
Sure